src/generics/middleware/authenticator.js

```diff
@@ -254,7 +254,8 @@ module.exports = async function (req, res, next, token = '') {
 			let fetchSingleOrgIdFunc = await fetchSingleOrgIdFromProvidedData(
 				decodedToken.data.tenant_id.toString(),
 				decodedToken.data.organization_ids,
-				req.headers['orgid']
+				req.headers['orgid'],
```

Comment:
orgid or orgId ?

Reply:
I added new param token so added the comma to this